components/ILIAS/Chatroom/classes/Bans/BannedUsersTable.php

```diff
@@ -57,6 +57,7 @@ public function getComponent(): UI\Component\Table\Data
         return $this->ui_factory
             ->table()
             ->data($this->lng->txt('ban_table_title'), $columns, $this)
+            ->withId(self::class)
```

Comment:
Interesting... The missing ID (even in ILIAS <= 9.x) can be considered as a bug, because the table is used every chat repository object.

Please add the `ref_id` of the chat room node here.